src/Game.cs

```diff
@@ -400,13 +439,29 @@ public void Run()
 			BeginRun();
 			gameTimer = Stopwatch.StartNew();
 
-			FNAPlatform.RunLoop(this);
+			RunLoop();
 
 			EndRun();
 
 			OnExiting(this, EventArgs.Empty);
 		}
 
+		private void RunLoop()
```

Comment:
Functionally okay, should be put in the `#region Private Methods` section for consistency.